pvnet_app/config.py

```diff
@@ -54,38 +54,146 @@ def overwrite_config_dropouts(config: dict) -> dict:
 
     Args:
         config: The data config
+
     """
 
     # Replace data sources
     for source in ["satellite"]:
         if source in config["input_data"]:
```

Comment:
Does this need to be nested twice? I think it could be the same as the NWP part below, like so: 
```suggestion
    if "satellite" in config["input_data"]:
```

Reply:
yea, that is nice - done